sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/orc/OrcTest.scala

```diff
@@ -78,7 +78,7 @@ abstract class OrcTest extends QueryTest with FileBasedDataSourceTest with Befor
       (f: String => Unit): Unit = withDataSourceFile(data)(f)
 
   /**
-   * Writes `data` to a Orc file and reads it back as a `DataFrame`,
+   * Writes `date` dataframe to a Orc file and reads it back as a `DataFrame`,
```

Comment:
Is there a reason you change this from `data` to `date`? This is not limited to `DATE`. The original one looks correct to me.

Reply:
Oops, a typo. :)